### ISSUE TITLE
Workers "visit" all at once, causing mass timeouts.

### ISSUE THREAD

**Issue by Aethios** [Noctem/Monocle#218]
I'm running 2000 workers with roughly 70,000 spawn points, 8% CPU usage and 70% RAM usage. Workers sit mostly idle, maybe 10% at a time operating normally, then suddenly all at once 90% of them will attempt to "visit" at the same time, causing a tsunami of timeouts. Any ideas?

**Comment by Aethios** [comment]
I should add that this only just started on 4/10. It started after I updated to the latest version via git pull that morning and turned off MORE_POINTS. Never had this problem before then.

**Comment by Jabsu** [comment]
I'm having hashing timeouts as well, but I've been having them almost two weeks now, so it's not because of the recent updates in my case. They happen every minute and last for a second. Otherwise everything seems to be working fine, Pokémon are being found fast enough etc. The spam is just a bit irritating. Don't know what's the problem, as my Internet connection seems to be stable.

Also, there's a lot of "visit cancelled" messages in my log, which is new.

(Spawns points: 1777, workers: 360 + 84)

**Comment by Aethios** [comment]
I don't think this is related to the "visit cancelled" errors. What shows in my log is "RPC request timed out" over and over and over.
